packages/health/example/lib/main.dart

```diff
@@ -160,14 +160,14 @@ class _HealthAppState extends State<HealthApp> {
         totalDistance: 2430,
         totalEnergyBurned: 400);
     success &= await health.writeBloodPressure(90, 80, earlier, now);
-    success &= await health.writeHealthData(
-        0.0, HealthDataType.SLEEP_REM, earlier, now);
-    success &= await health.writeHealthData(
-        0.0, HealthDataType.SLEEP_ASLEEP, earlier, now);
-    success &= await health.writeHealthData(
-        0.0, HealthDataType.SLEEP_AWAKE, earlier, now);
-    success &= await health.writeHealthData(
-        0.0, HealthDataType.SLEEP_DEEP, earlier, now);
+    // success &= await health.writeHealthData(
```

Comment:
Are these changes relevant to the PR?

Reply:
Yeah for some reason adding the sleep records is failing stating that it is an unsupported type. Although it is working on Android 13 and below.